core/kazoo_documents/src/kzd_ledgers.erl

```diff
@@ -293,6 +297,22 @@ set_dollar_amount(Doc, Amount) ->
     Units = kz_currency:dollars_to_units(Amount),
     kz_json:set_value(?AMOUNT, Units, Doc).
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec amount(doc()) -> kz_term:api_number().
+amount(Doc) ->
```

Comment:
@jamesaimonetti I was attempting to keep the requests for the amount to explicitly be for units or dollars, always storing units on the document so that the upstream code is more explicit.

Reply:
See other comment about breaking expectations